node/core/backing/src/lib.rs

```diff
@@ -521,12 +515,19 @@ impl CandidateBackingJob {
 					Ok(()) => (),
 				}
 			}
-			CandidateBackingMessage::GetBackedCandidates(_, tx) => {
+			CandidateBackingMessage::GetBackedCandidates(relay_parent, requested_candidates, tx) => {
 				let _timer = self.metrics.time_get_backed_candidates();
 
-				let backed = self.get_backed();
+				tracing::info!(target: LOG_TARGET, relay_parent = ?relay_parent, "started handling GetBackedCandidates");
+
+				let backed: Vec<_> = self
+					.get_backed()
+					.filter(|candidate| requested_candidates.contains(&candidate.hash()))
```

Comment:
This should use `table.get_candidate` as opposed to `table.proposed_candidates`.

Reply:
Within `get_backed`, you mean? I can do that, but it's different from what the code had previously been doing.

Reply:
Well, yes. The goal of the issue is to alter the code to not use `proposed_candidates` so we can give the provisioner more agency.

Reply:
Ok, f2059b8. Note that I used `attested_candidate` instead of `get_candidate`; I think that was your intent, but let me know if I was wrong.